src/device-registry/routes/api-v1.js

```diff
@@ -297,7 +297,6 @@ router.get(
 /***list devices */
 router.get(
```

Comment:
completely removing the cors middleware from a few GET endpoints for Sites., AirQlouds, and Devices.